lib/elixir_internal_certificate_web/controllers/user_search_controller.ex

```diff
@@ -13,10 +14,12 @@ defmodule ElixirInternalCertificateWeb.UserSearchController do
   def upload(conn, %{"file" => file}) do
     case CsvParsingHelper.validate_and_parse_keyword_file(file) do
       {:ok, keywords} ->
-        keywords_inserted = Scrapers.create_search_keyword(keywords, conn.assigns.current_user)
+        result = Scrapers.create_user_search(keywords, conn.assigns.current_user)
+        {keywords_count, uploaded_keywords} = result
+        JobQueueHelper.create_searching_queue(uploaded_keywords)
```

Comment:
should we move this step to inside the `Scrapers.create_user_search(keywords, conn.assigns.current_user)` method @nvminhtue, I think that should be a step in that method.

If we put it here, then the Worker won't be triggered if the Keyword is created in another way, for example, the other module also uploads the keyword, from the Iexs console or we have the API later,

Reply:
Fixed in 187c158